netdata.spec.in

```diff
@@ -248,13 +254,13 @@ export CFLAGS="${CFLAGS} -fPIC" && ${RPM_BUILD_DIR}/%{name}-%{version}/packaging
 autoreconf -ivf
 %configure \
 	%if 0%{!?_have_ebpf}
-	--disable-ebpf
+	--disable-ebpf \
 	%endif
 	%if 0%{!?_have_freeipmi}
-	--disable-plugin-freeipmi
+	--disable-plugin-freeipmi \
 	%endif
 	%if 0%{!?_have_nfacct}
-	--disable-plugin-nfacct
+	--disable-plugin-nfacct \
 	%endif
 	%if 0%{?centos_ver:1}
 	%if %{centos_ver} < 8
```

Comment:
You housekeep the old entries but you dont do anything to the systemd-plugin can't recall how the  _default autodetect_ works in the configure.ac https://github.com/netdata/netdata/commit/ce75313de06ee1e8aa5a59d57625ca25d467f0f8#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R72-R77

Reply:
If we find the libraries, it gets enabled.